Source/ImagePickerController.swift

```diff
@@ -301,26 +312,25 @@ extension ImagePickerController: ImageGalleryPanGestureDelegate {
   func panGestureDidChange(translation: CGPoint) {
     let galleryHeight = initialFrame.height - translation.y
```

Comment:
I tried this:
![screen shot 2015-10-14 at 22 58 28](https://cloud.githubusercontent.com/assets/8013017/10506253/ea38f658-7315-11e5-9249-059e2062752c.png)

So I guess, the solution in this PR right now is quite elegant and easy-to-read.
The only part which I can rewrite right now is early return, and rewriting it with `guard` will not make it any simpler.